css/FutureEventCardStyle.css

```diff
@@ -0,0 +1,58 @@
+.all {
```

Comment:
lets discuss during our meeting if we want the rectangular cards to have a rounded or sharp edge. I’m personally learning towards sharp edges because they look cleaner, but I’m curious to see what the rest of the team thinks